
public class Exception {
    public static void main(String args[]){
        int  numerador;
        float division;
        Integer denominador= null;
         numerador=20;
         
         try {
            division=numerador/denominador;  
         }catch(Exception e){
             System.out.println(e.getMessage);
             division=0;
             
         }
        
    }
    
}
